apps/src/javalab/JavalabConsole.jsx

```diff
@@ -243,7 +248,9 @@ export default connect(
     consoleLogs: state.javalab.consoleLogs,
     displayTheme: state.javalab.displayTheme,
     isPhotoPrompterOpen: state.javalab.isPhotoPrompterOpen,
-    photoPrompterPromptText: state.javalab.photoPrompterPromptText
+    photoPrompterPromptText: state.javalab.photoPrompterPromptText,
+    isRunning: state.javalab.isRunning,
+    isTesting: state.javalab.isTesting
```

Comment:
nit: could reduce this to a single boolean here so we don't have to add two new props (`canJumpToInput: state.javalab.isRunning || state.javalab.isTesting`)? it's very minor though, so i'm fine to ship as is :)

Reply:
good idea, updated